src/main/scala/services/mssql/MsSqlConnection.scala

```diff
@@ -236,8 +234,8 @@ object MsSqlConnection:
    * returned by the query if the result set is being closed without cancelling the statement first.
    * see: https://github.com/microsoft/mssql-jdbc/issues/877 for details.
    * ALL RESULT SETS CREATED FROM MS SQL CONNECTION MUST BE CLOSED THIS WAY
-   * @param resultSet The result set to close.
-   * @param statement The statement to close.
+   * resultSet The result set to close.
```

Comment:
I would suggest not removing the `@param` tag in scaladoc, it helps to render code docs properly and works:
<img width="879" alt="image" src="https://github.com/user-attachments/assets/7355433c-5b8e-4a75-b5bf-5eddbf250237" />

IDK why idea highlights it it seems to be a bug